src/pages/extending-agent-app/products-sdk/helpdesk-sdk/index.mdx

```diff
@@ -150,18 +150,40 @@ Listen for the ticket changes:
+| ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`                | Unique object identifier                                                                                                                             |
+| `shortId`           | Short ticket ID                                                                                                                                      |
+| `createdBy`         | Creator identifier                                                                                                                                   |
+| `updatedAt`         | Time of last modification                                                                                                                            |
+| `updatedBy`         | Modification author identifier                                                                                                                       |
+| `lastMessageAt`     | Time of last public message                                                                                                                          |
+| `events`            | Ticket events                                                                                                                                        |
+| `requester`         | Ticket requester                                                                                                                                     |
+| `status`            | Ticket status: `open`, `pending`, `onhold`, `solved`, `closed`                                                                                       |
+| `spam`              | Spam details                                                                                                                                         |
+| `subject`           | Ticket subject                                                                                                                                       |
+| `source`            | Ticket source                                                                                                                                        |
+| `tagIds`            | List of ticket tags                                                                                                                                  |
+| `teamIds`           | Teams that can access the ticket                                                                                                                     |
+| `assignment`        | Team / agent assignment                                                                                                                              |
+| `ratingRequestSent` | Has rating request been sent                                                                                                                         |
```

Comment:
@jakubsikora, quick question - I am not sure what this description means as it has a form of a question.🤔  Shouldn't it be `A rating request has been sent`?

Reply:
I only copied that from their documentation, but I think you are right :)